src/components/Comment/index.spec.tsx

```diff
@@ -24,22 +24,40 @@ describe(__filename, () => {
     Partial<DefaultProps>;
 
   const render = ({
-    store = configureStore(),
+    beginEdit = false,
     commentId = null,
+    fileName = null,
+    line = null,
+    store = configureStore(),
+    versionId = nextUniqueId(),
     ...moreProps
-  }: RenderParams = {}) => {
-    const props = {
-      addonId: 1,
+  }: RenderParams & { beginEdit?: boolean } = {}) => {
+    const keyParams = {
       commentId,
-      fileName: null,
-      line: null,
-      readOnly: false,
-      versionId: 2,
+      fileName,
+      line,
+      versionId,
+    };
+    if (beginEdit) {
+      store.dispatch(
+        actions.beginComment({
+          ...keyParams,
+          commentId: commentId || undefined,
+        }),
+      );
+    }
+    const props = {
```

Comment:
Nit: You might want to move `props` above the `if` statement for `beginEdit` because we tend to declare these things right at the top.